chef/cookbooks/postgresql/recipes/ha.rb

```diff
@@ -83,16 +83,17 @@
 
   pacemaker_colocation "col-#{service_name}" do
     score "inf"
-    resources [fs_primitive, vip_primitive, service_name]
+    resources "( #{fs_primitive} #{vip_primitive} ) #{service_name}"
     action :create
   end
 
   pacemaker_order "o-#{service_name}" do
     score "Mandatory"
-    ordering "#{fs_primitive} #{vip_primitive} #{service_name}"
+    ordering "( #{fs_primitive} #{vip_primitive} ) #{service_name}"
     action :create
     # This is our last constraint, so we can finally start service_name
     notifies :run, "execute[Cleanup #{service_name} after constraints]", :immediately
+    notifies :start, "pacemaker_primitive[#{vip_primitive}]", :immediately
```

Comment:
`#sleha` gurus just confirmed that starting `service_name` will not have any effect unless the previous resources in the ordering are already started.  So we *do* need this.